### PR TITLE
Update test procedure based on ONC feedback.

### PR DESCRIPTION
**Pull request by arscan**
This moves the OpenID Connect and Refresh Token tests within each of the launches test groups.  It also gets rid of the 'try two patients' test and related DB fields because we are just looking for the patient context on EHR launch now.

### CODE REVIEW

**Comment by arscan** [comment]
FYI @czh-orz -- please merge if it looks good to you.  Also, this touches a few files that you'll be editing when fixing the issue regarding launch / redirect URI not showing up in dev view, so you should make sure you build on top of this to avoid merge conflicts.

File: lib/app/modules/onc_module.yml [hunk 1]
```diff
@@ -46,14 +46,13 @@ test_sets:
           - SMARTDiscoverySequence
       - name: Standalone Patient App Launch
         overview: |
-          Servers must support the ability for an individual to access their medical record information
-          using an app that has been launched and authorized using the [SMART on FHIR Standalone Launch Sequence](https://www.hl7.org/fhir/smart-app-launch/#standalone-launch-sequence).
-          This test will initiate an authorization request and expects the [Patient Launch Context Parameter](https://www.hl7.org/fhir/smart-app-launch/scopes-and-launch-context/index.html#launch-context-arrives-with-your-access_token).
-          It will then perform a simple read of the Patient resource to ensure proper authorization.
+          Demonstrate the abiilty to perform an Patient Standalone Launch to a [SMART on FHIR](http://www.hl7.org/fhir/smart-app-launch/) client with a patient context,
```
Comment (czh-orz) [inline lib/app/modules/onc_module.yml]: Misspelled ability and should be "a Patient Standalone Launch"

File: lib/app/modules/onc_module.yml [hunk 2]
```diff
@@ -46,14 +46,13 @@ test_sets:
...
+          refresh token, and [OpenID Connect (OIDC)](https://openid.net/specs/openid-connect-core-1_0.html) identity token.  After launch, a simple Patient resource read is performed
+          on the patient in context.  The access token is then refreshed, and the Patient resource is read using the new
+          access token to ensure that the refresh was successful.  Finally, the authenication information provided by OpenID Connect is decoded and validated.
```
Comment (czh-orz) [inline lib/app/modules/onc_module.yml]: Misspelled "authentication"

File: lib/app/modules/onc_module.yml [hunk 3]
```diff
@@ -46,14 +46,13 @@ test_sets:
...
         input_instructions: |
-          Enter in the appropriate scope to enable patient-level access.  The test expects the Patient Launch Context Parameter to be returned that provides information about which patient has access.
-          
-          Note: *either this launch sequence or the EHR Practitioner App Launch test must also include scopes that demonstrate token refresh and OpenID Connect authentication.*
+          Enter in the appropriate scope to enable patient-level access to all relevant resources.  In addition, support for the 
+          OpenID Connect (`profile fhirUser`), refresh tokens (`offline_access`), and patient context (`launch/patient`) are required.
```
Comment (czh-orz) [inline lib/app/modules/onc_module.yml]: shouldn't `profile fhirUser` be `openid fhirUser`?

**Comment by arscan** [comment]
Thanks, fixed typos.  Anything else?

**Pull request merged**